src/Standards/PEAR/Sniffs/NamingConventions/ValidFunctionNameSniff.php

```diff
@@ -23,21 +23,23 @@ class ValidFunctionNameSniff extends AbstractScopeSniff
      * @var array
      */
     protected $magicMethods = [
-        'construct'  => true,
-        'destruct'   => true,
-        'call'       => true,
-        'callstatic' => true,
-        'get'        => true,
-        'set'        => true,
-        'isset'      => true,
-        'unset'      => true,
-        'sleep'      => true,
-        'wakeup'     => true,
-        'tostring'   => true,
-        'set_state'  => true,
-        'clone'      => true,
-        'invoke'     => true,
-        'debuginfo'  => true,
+        'construct'   => true,
+        'destruct'    => true,
+        'call'        => true,
+        'callstatic ' => true,
```

Comment:
A space character has been inserted at the end of the string `callstatic`, which is causing the failing unit tests.